upgrade/helpers/tasks.py

```diff
@@ -95,6 +95,8 @@ def sync_capsule_repos_to_upgrade(capsules):
     RHEV_CAPSULE_AK
         The AK name used in capsule subscription
     """
+    logger.info('Disabling the sync plan ...')
+    run('foreman-maintain advanced procedure run sync-plans-disable')
```

Comment:
We shouldnt do this - as we may miss issues related to sync plan migrations while enabled. Any special reason for doing this ?

Reply:
@jyejare https://bugzilla.redhat.com/show_bug.cgi?id=1721055#c7

Reply:
Ok, then we should bz_bug_is_open here.